docs/content/integrations/duckdb/reference.mdx

```diff
@@ -0,0 +1,584 @@
+---
+title: "dagster-duckdb integration reference"
+description: Store your Dagster assets in DuckDB
+---
+
+# dagster-duckdb integration reference
+
+This reference page provides information for working with [`dagster-duckdb`](/\_apidocs/libraries/dagster-duckdb) features that are not covered as part of the [Using Dagster with DuckDB tutorial](/integrations/duckdb/using-duckdb-with-dagster).
+
+- [Selecting specific columns in a downstream asset](#selecting-specific-columns-in-a-downstream-asset)
+- [Storing partitioned assets](#storing-partitioned-assets)
+- [Storing tables in multiple schemas](#storing-tables-in-multiple-schemas)
+- [Using the DuckDB I/O manager with other I/O managers](#using-the-duckdb-io-manager-with-other-io-managers)
+- [Storing and loading PySpark and Polars DataFrames in DuckDB](#storing-and-loading-pyspark-and-polars-dataframes-in-duckdb)
```

Comment:
The heading for this section is `PySpark OR Polars`, whereas this is `AND`

I don't have a preference which one you use, just pointing this out so you can fix the link!